plugins/search/src/components/SearchContext/index.tsx

```diff
@@ -14,4 +14,8 @@
  * limitations under the License.
  */
 
-export { SearchContextProvider, useSearch } from './SearchContext';
+export {
+  SearchContextProvider,
+  SearchContext,
+  useSearch,
```

Comment:
Think this needs a changeset? 👍

Reply:
@benjdlambert is a changeset needed if it's not exported from the package?  Believe it's only used in the `.storybook.tsx` files in the folder.